src/widget/wtrackmenu.cpp

```diff
@@ -558,6 +563,8 @@ void WTrackMenu::loadTracks(TrackIdList trackIdList) {
     // Clean all forms of track store
     clearTrackSelection();
 
+    DEBUG_ASSERT(!m_pTrackModel);
```

Comment:
It's confusing, but `DEBUG_ASSERT` works backwards from `VERIFY_OR_DEBUG_ASSERT`, so remove the `!` here.

Reply:
It's meant to assert that this function should only be called if a TrackModel has not been set in the constructor. Using this function with TrackModel will lead to inconsistent behaviour.

Reply:
`DEBUG_ASSERT` is triggered if the condition is true; `VERIFY_OR_DEBUG_ASSERT` is triggered if the condition is false.

Reply:
Okay. Will change this VERIFY_OR_DEBUG_ASSERT to maintain consistency.

Reply:
`VERIFY_OR_DEBUG_ASSERT` is used when there is a fallback action to execute. In this case, the appropriate fallback is to continue executing the rest of the function, so `DEBUG_ASSERT` is more appropriate.

Reply:
`DEBUG_ASSERT` makes sure its parameter is a truthy value. I checked it explicitly by writing
```cpp
DEBUG_ASSERT(true);
DEBUG_ASSERT(false);
```
and it warned at `false`.
This line from `assert.h` seems to tell the same.
```cpp
#define DEBUG_ASSERT(cond) ((!(cond)) ? mixxx_debug_assert(#cond, __FILE__, __LINE__, ASSERT_FUNCTION) : mixxx_noop())
```

Reply:
Okay, sorry for confusing you.

Reply:
No issues :)